cli/compose/loader/full-example.yml

```diff
@@ -280,6 +280,15 @@ volumes:
       foo: "bar"
       baz: 1
 
+  another-volume:
+    name: "user_specified_name"
+    driver: vsphere
+
+    driver_opts:
+      # Values can be strings or numbers
+      foo: "bar"
+      baz: 1
+
   external-volume:
     # Specifies that a pre-existing volume called "external-volume"
     # can be referred to within this file as "external-volume"
```

Comment:
for `other-external-volume` below, should we update the comment to mention `volume.external.name` is deprecated? (can't comment on that line, so leaving the comment here :smile:)

```
# This example uses the deprecated "volume.external.name" (replaced by "volume.name")
```